src/config/Config.ts

```diff
@@ -43,8 +44,6 @@ export default class Config implements ConfigInterface {
   }
 
   public set(field: string, value: string[]) {
-    if (!this.has(field)) return;
```

Comment:
I'd rather keep this line to prevent fields that do not exist to the config, as it's written directly to disk 😅

Reply:
I removed this because I saw that the `set` function was responsible for validating and writing config. After checking the callees of the function (configCommand) I thought we could be better keeping this out as the config command also calls the `has()` to validate for valid input.

Reply:
I'd rather keep it.

`ConfigCommand` has additional output, therefore it uses both `has` and `set`.

Reply:
I'm getting into a bit of a fight with this as some values that I'm setting from the environment variables are not in the `MODIFIABLE_FIELDS` e.g. the token and client id :). Would it be okay to change this with a check for valid `JSON_KEYS`? 😄